fdmi/plugins/motr-fdmi-app.md

```diff
@@ -20,108 +20,102 @@ The FDMI source instances send matched records to the FDMI Application or Plugin
+
 
 ## How to create and test other FDMI Applications with examples
 
 All FDMI related service code and examples are in the `cortx-motr/fdmi` directory in the following link: [fdmi dir]( https://github.com/Seagate/cortx-motr/tree/main/fdmi)
 
 All the FDMI demo documents are available in `cortx-motr/doc/fdmi-demo` directory. [Click here]( https://github.com/Seagate/cortx-motr/tree/main/doc/fdmi_demo/demo-fdmi) to check. 
 
-One of the FDMI Application example is within the `plugins` directory named as [fdmi_sample_plugin]( https://github.com/Seagate/cortx-motr/blob/main/fdmi/plugins/fdmi_sample_plugin.c). 
+One of the FDMI Application example is within the `plugins` directory named as [fdmi_sample_plugin]( https://github.com/Seagate/cortx-motr/blob/main/fdmi/plugins/fdmi_sample_plugin.c).
 
 This application is a FDMI sample plugin written in C using `motr/client.h` interface which connects to the cluster (FDMI source instance) and registers a listener to listen for Key-Value events matching the specific FDMI filter substrings. This application then merely prints to standard output the matched records.
 
 The executable binary file will be compiled as part of the initial Motr compilation using the corresponding `Makefile.am` dependency and the [Makefile.sub]( https://github.com/Seagate/cortx-motr/blob/main/fdmi/plugins/Makefile.sub) file defined in the same folder.
 
-The `fdmi_sample_plugin` application can be tested in two forms: 
+The `fdmi_sample_plugin` application can be tested in two forms:
 
 - Running the `fdmi_app` python script
 - Running the `fdmi_plugin_st` shell script
 
-For the first case, `fdmi_sample_plugin` communicates with the [fdmi_app]( https://github.com/Seagate/cortx-motr/blob/main/fdmi/plugins/fdmi_app) python script by printing to standard output all the FDMI records. 
+For the first case, `fdmi_sample_plugin` communicates with the [fdmi_app]( https://github.com/Seagate/cortx-motr/blob/main/fdmi/plugins/fdmi_app) python script by printing to standard output all the FDMI records.
 
 Before testing with the `fdmi_app`, we need to have a CORTX-Motr cluster already up and running with FDMI capabilities by following these steps:
 
--  First, deploy a CORTX-Motr using specific configuration file and the instructions provided in the cortx-hare repo following this link: 
+-  First, deploy a CORTX-Motr using specific configuration file and the instructions provided in the cortx-hare repo following this link:
 [CORTX deployment]( https://github.com/Seagate/cortx-hare/blob/main/README.md)
 
--  Second, add FDMI capabilities by defining and customizing the fdmi_filters subsection in the `CDF.yaml` config file used for deploying with `hctl` command. 
-Example of a config file with the needed subsection created can be found in the following link: 
+-  Second, add FDMI capabilities by defining and customizing the fdmi_filters subsection in the `CDF.yaml` config file used for deploying with `hctl` command.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/Seagate/cortx-motr/pullRequest?prid=9216996)
```suggestion
- Second, add FDMI capabilities by defining and customizing the fdmi_filters subsection in the `CDF.yaml` config file used for deploying with `hctl` command. 
```